_posts/2016-06-05-incremental-property-based-testing.md

```diff
@@ -249,9 +249,12 @@ Going forward
 
 We stopped there, but we need not have. Just as the test should have held for
 any branch, it should also hold for any repository. We were just creating an
-empty repository because it was convenient for us.
+empty repository because it fit our focus here. [Hypothesis' data generation
+toolkit can construct instances of non-primitive data types](
+{% post_url 2016-05-11-generating-the-right-data %}) like our `Repository`
+class.
 
-If we were to continue, the test would have looked something like this:
+The next step would be to write a `repositories` strategy for our test:
```

Comment:
I find the follow-on doesn't quite work here - the way this is phrased suggests it's about to show how to do that, but instead it just shows an instance of using it. Maybe rephrase?